dev/integration_tests/android_semantics_testing/lib/main.dart

```diff
@@ -23,18 +23,18 @@ void main() {
 
 const MethodChannel kSemanticsChannel = MethodChannel('semantics');
 
-Future<String> dataHandler(String message) async {
-  if (message.contains('getSemanticsNode')) {
+Future<String> dataHandler(String? message) async {
+  if (message!.contains('getSemanticsNode')) {
```

Comment:
Will this be an error here if `message` is null or am I reading this wrong?  Or if `message` will never be null, then can't we just keep it as `String` instead of `String?`?

Reply:
Good point. I went ahead and added a null check here, but not sure it was that critical, as if it is null we are just going to throw an `UnimplementedError` error anyway, but this is probably a better result than a null check error.